quickwit-cli/src/lib.rs

```diff
@@ -221,7 +233,7 @@ pub async fn index_data_cli(args: IndexDataArgs) -> anyhow::Result<()> {
         start_statistics_reporting_loop(supervisor_handler, args.input_path.clone()).await?;
 
     if statistics.num_published_splits > 0 {
-        println!("You can now query your index with `quickwit search --index-uri {} --query \"barack obama\"`" , args.index_uri);
+        println!("You can now query your index with `quickwit search --index-id {} --query \"barack obama\"`" , args.index_id);
```

Comment:
Is that gonna work? There's no metastore URI in the command.

Reply:
nice catch, we need the metastore uri.